ReactSkia/views/common/RSkTextUtils.cpp

```diff
@@ -0,0 +1,89 @@
+ /*
+ * Copyright (C) 1994-2021 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "ReactSkia/views/common/RSkTextUtils.h"
+
+using namespace skia::textlayout;
+
+namespace facebook {
+namespace react {
+namespace {
+inline int getLines(std::vector<LineMetrics>& metrics, float frameHeight) {
+    int i=0;
+    int minNumberOfLines = 0;
+    for(; i< metrics.size(); i++) {
```

Comment:
nit: will it be better to have int i declaration in for loop, for redability

Reply:
Updated